nova/virt/vmwareapi/driver.py

```diff
@@ -1270,52 +1270,57 @@ def in_cluster_vmotion(self, context, instance, host_moref_value):
         We do not need to check if instance is part of our cluster, because we
         only get called if the instance belongs to us - in the DB at least.
         """
-        host_ref = vim_util.get_moref(host_moref_value, 'HostSystem')
```

Comment:
you _could_ introduce a `def _in_cluster_vmotion()` method here which you could wrap, which would at least reduce the diff considerably.

Reply:
Yes, I could considered that. It doesn't have any advantage over using `EventReporter` directly imho.

We'll probably merge this commit with the original introduction of the feature in the upcoming upgrade of Nova anyways, so I do not consider the diff-size relevant.